### PR TITLE
Release 0.3.0

### PR DESCRIPTION
**Pull request by JeanTracker**
NuguSDK release 0.3.0

Signed-off-by: Hyojoong Kim <hyojoong.kim@sk.com>

### CODE REVIEW

File: packaging/bionic/changelog [hunk 1]
```diff
@@ -1,3 +1,12 @@
+nugu (0.3.0ubuntu1~bionic) bionic; urgency=medium
+
+  * Applied updated version of capability interface
+  * Added delegation interface
+  * Added directive versioning
+  * Fixed static analysis issues
+
+ -- nugulinux <nugulinux@gmail.com>  Thu, 7 Dec 2019 12:00:00 +0900
```
Comment (kimhyungrok) [inline packaging/bionic/changelog]: typo error. Please change "Dec" to "Nov"

File: packaging/xenial/changelog [hunk 2]
```diff
@@ -1,3 +1,12 @@
+nugu (0.3.0ubuntu1~xenial) xenial; urgency=medium
+
+  * Applied updated version of capability interface
+  * Added delegation interface
+  * Added directive versioning
+  * Fixed static analysis issues
+
+ -- nugulinux <nugulinux@gmail.com>  Thu, 7 Dec 2019 12:00:00 +0900
```
Comment (kimhyungrok) [inline packaging/xenial/changelog]: typo error. Please change "Dec" to "Nov"

File: packaging/bionic/changelog (hunk 1 above)
Comment (JeanTracker) [reply packaging/bionic/changelog]: thanks.

File: packaging/xenial/changelog (hunk 2 above)
Comment (JeanTracker) [reply packaging/xenial/changelog]: thanks.

**Pull request merged**